src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java

```diff
@@ -1,617 +1,665 @@
 package org.mariadb.jdbc.internal.stream;
 
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
 import java.sql.Timestamp;
 import java.util.Calendar;
 import java.util.zip.DeflaterOutputStream;
 
+import org.mariadb.jdbc.internal.util.ByteBuffer;
 
 public class PacketOutputStream extends OutputStream {
```

Comment:
that was about whitespace changes in the source files. I agree with mike, it is not possible to see what really changed , becase the diff is in every line